crates/diesel_models/src/payment_intent.rs

```diff
@@ -39,40 +34,45 @@ pub struct PaymentIntent {
     pub off_session: Option<bool>,
```

Comment:
This `off_session` we shouldn't use, instead our logic will be based on `apply_mit_exemption`